sigoptlite/sources.py

```diff
@@ -56,8 +56,11 @@ def remove_transformations_from_source_suggestion(self, source_suggestion):
     return self.remove_conditional_transformation_from_suggestion(self._original_experiment, source_suggestion)
 
   def get_suggestion(self, observations):
-    suggested_point, suggested_task_cost = self.next_point(observations)
-    assignments = self.make_assignments_from_point(self.experiment, suggested_point)
+    suggested_points, suggested_task_cost = self.next_point(observations)
+    if len(suggested_points) == 0:
+      raise SigoptComputeError("Was unable to generate a suggestion")
```

Comment:
Something more specific like `GP Source exhausted unique suggestions`?

Reply:
Are we sure that this is the only way GP source can't generate a suggestion?

Reply:
Oh okay, never mind I understand... alright